boards/arm/cc1352p1_launchxl/cc1352p1_launchxl.dts

```diff
@@ -0,0 +1,147 @@
+/*
+ * Copyright (c) 2021 Florin Stancu
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+
+#include <ti/cc1352r.dtsi>
+#include "../cc1352r1_launchxl/boosterpack_connector.dtsi"
+#include "cc1352p1_launchxl-pinctrl.dtsi"
+
+/ {
+	model = "TI CC1352P1 LaunchXL";
+	compatible = "ti,launchxl-cc1352p1";
+
+	aliases {
+		led0 = &led0;
+		led1 = &led1;
+		sw0 = &btn0;
+		sw1 = &btn1;
+	};
+
+	chosen {
+		zephyr,sram = &sram0;
+		zephyr,flash = &flash0;
+		zephyr,console = &uart0;
+		zephyr,shell-uart = &uart0;
+		zephyr,ieee802154 = &ieee802154;
+	};
+
+	/**
+	 * The CC1352P LAUNCHXL has an on-board antenna switch (SKY13317-373LF) used to select the
+	 * appropriate RF signal port based on the currently-used PHY.
+	 *
+	 * Truth table:
+	 *
+	 * Path        DIO28 DIO29 DIO30
+	 * =========== ===== ===== =====
+	 * Off         0     0     0
+	 * Sub-1 GHz   0     0     1     // DIO30 mux to IOC_PORT_RFC_GPO0 for auto
+	 * 2.4 GHz     1     0     0     // DIO28 mux to IOC_PORT_RFC_GPO0 for auto
+	 * 20 dBm TX   0     1     0     // DIO29 mux to IOC_PORT_RFC_GPO3 for auto
+	 */
+	antenna_mux0: antenna_mux0 {
+		compatible = "ti,antenna-mux";
```

Comment:
antenna switch (SKY13317-373LF), so compatible looks wrong.

Reply:
True, renamed binding to `skyworks,sky13317` (kept private to `cc1352p` board)